homeassistant/components/ecobee/climate.py

```diff
@@ -304,6 +308,16 @@ def resume_program_set_service(service: ServiceCall) -> None:
         "set_occupancy_modes",
     )
 
+    platform.async_register_entity_service(
+        SERVICE_SET_ACTIVE_SENSORS,
+        {
+            vol.Required(ATTR_ENTITY_ID): cv.entity_id,
```

Comment:
```suggestion
```

Will be automatically added by `async_register_entity_service`

Reply:
Removed the erroneous line.